hoodie-client/src/main/java/com/uber/hoodie/io/HoodieUpdateHandle.java

```diff
@@ -93,6 +94,9 @@ private void init(String fileId, Iterator<HoodieRecord<T>> newRecordsItr) {
                     oldFilePath = new Path(
                         config.getBasePath() + "/" + record.getPartitionPath() + "/"
                             + latestValidFilePath);
+                    relativePath = new Path(
```

Comment:
can this be more terse?  

```
String relativePathStr = record.getPartitionPath() + "/" + FSUtils
 +                            .makeDataFileName(commitTime, TaskContext.getPartitionId(), fileId);

newFilePath = new Path(.. + relativePathStr) 

.. 

writeStatus.getStat().setRelativePath(relativePathStr)?
```

Also `relativePathStr` can simply be a local variable correct?